### PR TITLE
feat: Capture and pass currency param

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: app/script/auth/localeConfig.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import {getURLParameter} from './util/urlUtil';
 export const SUPPORTED_LANGUAGE = require('./supportedLocales');
 
 export const DEFAULT_LANGUAGE = 'en';
+export const DEFAULT_CURRENCY = 'eur';
```
Comment (bennycode) [inline app/script/auth/localeConfig.js]: Sorting... `DEFAULT_CURRENCY` comes before `DEFAULT_LANGUAGE`

File: app/script/auth/localeConfig.js [hunk 2]
```diff
@@ -13,3 +14,8 @@ export function currentLanguage() {
   const languageParam = getURLParameter('hl');
   return languageParam || getLocale().substr(0, LANGUAGE_SHORTHAND_LENGTH) || DEFAULT_LANGUAGE;
 }
+
+export function currentCurrency() {
+  const currencyParam = getURLParameter('currency');
+  return currencyParam || DEFAULT_CURRENCY;
```
Comment (bennycode) [inline app/script/auth/localeConfig.js]: Could be just `return getURLParameter('currency') || DEFAULT_CURRENCY;`

File: app/script/auth/localeConfig.js [hunk 3]
```diff
@@ -10,6 +11,9 @@ function getLocale() {
 
 export function currentLanguage() {
   const LANGUAGE_SHORTHAND_LENGTH = 2;
-  const languageParam = getURLParameter('hl');
-  return languageParam || getLocale().substr(0, LANGUAGE_SHORTHAND_LENGTH) || DEFAULT_LANGUAGE;
+  return getURLParameter('hl') || getLocale().substr(0, LANGUAGE_SHORTHAND_LENGTH) || DEFAULT_LANGUAGE;
+}
+
+export function currentCurrency() {
+  return getURLParameter('currency') || DEFAULT_CURRENCY;
```
Comment (gregor) [inline app/script/auth/localeConfig.js:18]: Time to build an enum with the URL params?

Comment (bennycode) [reply app/script/auth/localeConfig.js:18]: Good idea!

**Pull request merged**